docs/proposals/connector-protocol-v2.md

```diff
@@ -87,12 +88,10 @@ Type specs:
 ```erlang
 -type frame() :: iolist().
 -type point_of_reference() ::
-    non_neg_integer() |
-    binary() |
-    undefined.
+    non_neg_integer().
 ```
 
-Constructors:
+Constructors follow.  Note that all integers larger than 8 bits are encoded in little endian byte order.
```

Comment:
If we use BE for the outer frame, we should probably be consistent and use BE here as well. I've used the shorthand functions to make it easy to swap the integer encodings. If we don't change the functions then the frame function should be corrected to use a u23be variant.

Reply:
I was being swayed by your memory map case, but since we aren't being 100% picky about byte alignment, I'll change everything to big endian.